### PR TITLE
expose gh base url

### PR DESCRIPTION
**Pull request by MosheEichler**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/40279

## Description
Expose GitHub URL to allow customers set it up themselves

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GitHub/Integrations/GitHub/GitHub.py [hunk 1]
```diff
@@ -2001,6 +2002,7 @@ def main():
     handle_proxy()
     cmd = demisto.command()
     LOG(f'command is {cmd}')
+    demisto.log(BASE_URL)
```
Comment (MosheEichler) [inline Packs/GitHub/Integrations/GitHub/GitHub.py:2005]: To can fetch it in the UT
Screen Shot 2021-08-12 at 13 45 15

Comment (teizenman) [reply Packs/GitHub/Integrations/GitHub/GitHub.py:2005]: This is not the way to do that.
You should find a way to do the UT without adding code that is not relevant for the integration.

Comment (MosheEichler) [reply Packs/GitHub/Integrations/GitHub/GitHub.py]: I fixed it. It's simpler than I thought

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 2]
```diff
@@ -16,6 +16,11 @@ configuration:
   - Pull_requests
   required: false
   type: 15
+- display: URL
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:19]:
```suggestion
- display: Server URL
```

File: Packs/GitHub/ReleaseNotes/1_3_6.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### GitHub
+ - Expose *URL* parameter to allow users set it up themselves
```
Comment (yaakovi) [inline Packs/GitHub/ReleaseNotes/1_3_6.md:4]:
```suggestion
- Added the **Server URL** integration parameter.
```

**Review by yaakovi — APPROVED** [review]

**Review by teizenman — APPROVED** [review]

**Pull request merged**